README.md

```diff
@@ -75,6 +75,8 @@ $ pip install -r docs/requirements.txt
 - Ekaterina Tsvetoslavova Stankulova ([Center for Advanced Systems Understanding](https://www.casus.science/))
 
 ## Citing atoMEC
-The following paper should be cited in publications which use atoMEC:
+If you use code from this repository in a published work, please cite the DOI corresponding to the specific version of atoMEC that you used (DOIs are listed at [Zenodo.org](https://doi.org/10.5281/zenodo.5205718)).
+
+Furthermore we kindly ask you to cite also the following paper in publications which use atoMEC:
```

Comment:
```suggestion
If appropriate, you might also consider citing the following preprint in publications which use atoMEC:
```
Probably it's enough for most people just to cite the code, the preprint reference was there in the absence of a code reference. But there are some cases in which people ought to cite both so it's useful to keep the preprint ref in some form.